### PR TITLE
BUG: fix handling of NaNs in volatility plot

### LINKED ISSUE #175: Nan Handling in Feature-Volatility Plot Bug
**Bug Description**

Screen Shot 2022-10-04 at 10 36 16 AM

This is the same issue as https://github.com/qiime2/q2-feature-table/issues/259. 
It was fixed by this pull request: https://github.com/qiime2/q2-feature-table/pull/260, the same changes need to be applied to the feature-volatility visualization

### PR DESCRIPTION
**Pull request by lizgehret**
Fixes #175

### CODE REVIEW

**Comment by lizgehret** [comment]
Will add a unit test that invokes this behavior and asserts that NULL values are found within the JSON (vs. NaN)

**Comment by lizgehret** [comment]
Co-Authored by: @colinvwood

File: q2_longitudinal/_utilities.py [hunk 1]
```diff
@@ -814,6 +814,7 @@ def _summarize_feature_stats(table, state_md_col):
 # df.fillna(None) does not work so used solution from:
 # https://github.com/pandas-dev/pandas/issues/1972
 def _convert_nan_to_none(df):
+    df = df.astype(object)
```
Comment (Oddant1) [inline q2_longitudinal/_utilities.py:817]: I just fixed another issue caused by this same kinda thing [here](https://github.com/qiime2/q2-types/pull/294). Annoying af

**Review by Oddant1 — APPROVED** [review]
lgtm
